<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiPlayerScreen</class>
 <widget class="QWidget" name="MultiPlayerScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="JoinStartPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="TitleLabel">
         <property name="styleSheet">
          <string notr="true">font-size: 30px;</string>
         </property>
         <property name="text">
          <string>Multiplayer</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="ErrorLabel">
         <property name="styleSheet">
          <string notr="true">font-size: 30px;
color: rgb(239, 41, 41);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLineEdit" name="HostInput"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="PortInput"/>
         </item>
         <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignBottom">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Host IP</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignBottom">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="StartServerButton">
         <property name="text">
          <string>Start Server</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="JoinServerButton">
         <property name="text">
          <string>Join Server</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="MainMenuButton">
         <property name="text">
          <string>Main Menu</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ServerHostPage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Some Ready shit here</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPlainTextEdit" name="ChatBox">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="1" column="0">
          <widget class="QLineEdit" name="LowestInput"/>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="AmountInput"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="HighestInput"/>
         </item>
         <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignBottom">
          <widget class="QLabel" name="LowestLabel">
           <property name="text">
            <string>Lowest</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignBottom">
          <widget class="QLabel" name="HighestLabel">
           <property name="text">
            <string>Highest</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" alignment="Qt::AlignHCenter|Qt::AlignBottom">
          <widget class="QLabel" name="AmountLabel">
           <property name="text">
            <string>Amount</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QPushButton" name="StartGameButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Start Game</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
